src/main/java/org/minions/devfund/royrodriguez/WhackAMole.java

```diff
@@ -0,0 +1,145 @@
+package org.minions.devfund.royrodriguez;
+
+import java.util.Arrays;
+import java.util.Random;
+
+/**
+ * Whack a Mole game.
+ */
+public class WhackAMole {
+
+    private char[][] moleGrid;
+    private int score;
+    private int attemptsLeft;
+    private int molesLeft;
+    private static final char MOLE = 'M';
+    private static final char WHACKED = 'W';
+    private static final char EMPTY_PLACE = '*';
+
+    /**
+     * Constructor.
+     *
+     * @param numAttempts   max. attempts.
+     * @param gridDimension dimensions.
+     */
+    public WhackAMole(int numAttempts, int gridDimension) {
+        this.attemptsLeft = numAttempts;
+        score = 0;
+        molesLeft = 0;
```

Comment:
These lines of code are redundant.
integer attributes of class are zero by default.